wrapper/src/main/resources/messages.properties

```diff
@@ -164,15 +167,18 @@ MonitorThreadContainer.emptyNodeKeys=Provided node keys are empty.
 MonitorImpl.contextNullWarning=Parameter 'context' should not be null.
 
 # Monitor Service Impl
-MonitorServiceImpl.nullMonitorParam=Parameter monitor' should not be null.
-MonitorServiceImpl.emptyAliasSet=Empty alias set passed for {0}. Set should not be empty.
+MonitorServiceImpl.nullMonitorParam=Parameter 'monitor' should not be null.
+MonitorServiceImpl.emptyAliasSet=Empty alias set passed for ''{0}''. Set should not be empty.
+MonitorServiceImpl.errorPopulatingAliases=Error occurred while populating aliases: ''{0}''.
 
 # Plugin Service Impl
 PluginServiceImpl.hostListEmpty=Current host list is empty.
 PluginServiceImpl.releaseResources=Releasing resources.
 PluginServiceImpl.hostListException=Exception while getting a host list.
 PluginServiceImpl.hostAliasNotFound=Can''t find any host by the following aliases: ''{0}''.
 PluginServiceImpl.hostsChangelistEmpty=There are no changes in the hosts' availability.
+PluginServiceImpl.failedToRetrieveHostPort=Could not retrieve Host:Port for connection.
+PluginServiceImpl.nonEmptyAliases=fillAliases called when HostSpec already contains the following aliases: ''{0}''. Please reset HostSpec aliases before calling this method.
```

Comment:
"Please reset HostSpec aliases before calling this method."

I think it may mislead developers. 
"Please reset HostSpec aliases if you need to re-fill them again." ?